templates/CSI/cephfs/pvc.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
The same for PVC, no reason to maintain 2 pvs yamls for RBD and CephFS

Reply:
ack. will change it.